doc/api/test.md

```diff
@@ -352,6 +352,62 @@ Otherwise, the test is considered to be a failure. Test files must be
 executable by Node.js, but are not required to use the `node:test` module
 internally.
 
+## Mocking
+
+The `node:test` module supports mocking during testing via a top-level `mock`
+object. The following example creates a spy on a function that adds two numbers
+together. The spy is then used to assert that the function was called as
+expected.
+
+```js
+test('spies on a function', () => {
+  const sum = test.mock.fn((a, b) => {
```

Comment:
```suggestion
require { test, mock } from 'node:test'
test('spies on a function', () => {
  const sum = mock.fn((a, b) => {
```
I think this better demonstrates this new ability can be used with any testing FW

Reply:
Fair enough. I was just trying to avoid a code sample that needed both CJS and ESM implementations 😄